parser-typechecker/src/Unison/PrintError.hs

```diff
@@ -375,13 +375,14 @@ renderTypeError e env src curPath = case e of
             -- , (,Color.ForceShow) <$> rangeForType foundType
             -- , (,Color.ForceShow) <$> rangeForType expectedType
             -- ,
-            (,Type1) <$> rangeForAnnotated mismatchSite,
+            (,Type1) . startingLine <$> (rangeForAnnotated mismatchSite),
```

Comment:
The mismatchSite for a `let` block includes the entire remainder of the block, which is excessive. Clipping this to just the first line.